src/app/components/not-found/not-found.component.html

```diff
@@ -1,3 +1,8 @@
-<p>
-  not-found works!
-</p>
+<div class="p-grid">
+  <div class="p-col-12 p-md-6 p-lg-4"> </div>
+  <div class="p-col-12 p-md-6 p-lg-4">
+    <img src="assets/common/logos/not-found.jpg" width="100%">
+    <h1>{{ 'NOTFOUND_NICE' | translate }}</h1>
+  </div>
+  <div class="p-col-12 p-md-6 p-lg-4"></div>
+</div>
```

Comment:
Im NotFound Component muss ich immer scrollen um den Text unter dem Bild zu sehen, es wäre besser wenn es an der Bildschirmgröße anpassen würde

Reply:
ich weiß nur noch nicht wie das geht, ich recherchiere mal

Reply:
Es gibt die Einheit vh (viewport height) in css die man vielleicht benutzen könnte (1vw = 1% der viewport height)

Reply:
das bekomme ich heute nicht mehr hin. ich lass es mal wie es ist für morgen und mach ein task in jira für die anpassung an bildschirmgröße

Reply:
erledigt, war doch einfacher als gedacht ,-) allerdings musste ich 300vh nehmen.. anscheinend sind das pixel, nicht prozent ;-)